docs/hosting.md

```diff
@@ -2,4 +2,91 @@
 
 The process of hosting a Robyn app on various cloud providers.
 
-Coming Soon....
+
+### Railway
+
+To deploy an app on Railway is necessary to have a Github account, because the apps are deploying from a Github repository. 
+
+The repository we will be deploy is a simple app that display a "Hello World" message and serve a HTML file.
+
```

Comment:
Can we write the introduction as 

We will be deploying the app on [Railway](https://railway.app/).

**A `GitHub` account is needed as a mandatory pre-requisite.**

We will be deploying a sample "Hello World", which will be demonstrating a simple `GET` route and serving an HTML file.